2021q4/boot-performance.adoc

```diff
@@ -0,0 +1,29 @@
+=== Boot Performance Improvements
+
+Links: +
+link:https://wiki.freebsd.org/BootTime[Wiki page] URL: link:https://wiki.freebsd.org/BootTime[https://wiki.freebsd.org/BootTime] +
+link:https://www.daemonology.net/blog/2021-08-12-EC2-boot-time-benchmarking.html[OS boot time comparison] URL: link:https://www.daemonology.net/blog/2021-08-12-EC2-boot-time-benchmarking.html[https://www.daemonology.net/blog/2021-08-12-EC2-boot-time-benchmarking.html]
+
+Contact: Colin Percival <cperciva@FreeBSD.org>
+
+Colin Percival is coordinating an effort to speed up the FreeBSD boot process.
+For benchmarking purposes, he is primarily using an EC2 c5.xlarge instance as a
+reference platform and is measuring the time between when the virtual machine
+enters the EC2 "running" state and when it is possible to SSH into the instance.
+
+This work started in 2017, and as of the end of September 2021 the FreeBSD boot
+time was reduced from approximately 30 seconds to approximately 15 seconds.
```

Comment:
should we clarify what this includes? power on to login prompt?

Reply:
The measurement is from the EC2 instance entering the "running" state to when we can SSH in.  I think "running" means roughly "CPU reset" but I'm not 100% certain (and Amazon doesn't discuss those sorts of internals).

Happy to add the detail but I'm not entirely sure that it matters; this detail is on the wiki page if anyone is interested enough to click on links.